commons/files.go

```diff
@@ -8,7 +8,8 @@ import (
 	"regexp"
 )
 
-var hl7SplitToken = regexp.MustCompile("(\\r(\\n|\\x1c)+(\\n\\r)?|$)")
+var hl7SplitToken = regexp.MustCompile("(\\r(\\n|\\x1c)+(\\n\\r)?MSH\\||$)")
+var hl7FindStartToken = regexp.MustCompile("(MSH\\||$)")
 
 const scanBufferSize = 10 * 1024 * 1024
```

Comment:
Increase buffer to 64 * 1024 * 1024?

Reply:
no that's not the issue. there is no record that exceeds 10mb. this just punts to a longer length. the real issue is that this does not like records that do not end in the buffer? which will happen for any file longer than the buffer (like most epic files are, even compared to 64mb?)

Reply:
you can test this yourself by increasing the buffer to what you suggest and then processing a file bigger than that?